components/interactive_dialog/dialog_introduction_text.test.jsx

```diff
@@ -6,7 +6,7 @@ import {mount} from 'enzyme';
 
 import EmojiMap from 'utils/emoji_map';
 
-import DialogIntroductionText from './dialog_introduction_text.jsx';
+import DialogIntroductionText from './dialog_introduction_text.tsx';
 
```

Comment:
Thanks @VishakhaPoonia would it be possible to migrate the test to typescript as well? Just by passing the prop types to the test as well.